timeseries/tests/unittests/common.py

```diff
@@ -185,9 +197,12 @@ def clear_past_metrics(self) -> None:
         del self._past_target_mean
 
 
-def get_prediction_for_df(data):
-    predictor = TimeSeriesPredictor(prediction_length=5).fit(data, hyperparameters={"Naive": {}})
-    return predictor.predict(data)
+def get_prediction_for_df(data, prediction_length=5):
+    forecast_index = get_forecast_horizon_index_ts_dataframe(data, prediction_length=prediction_length)
+    columns = ["mean", "0.1", "0.5", "0.9"]
+    return TimeSeriesDataFrame(
+        pd.DataFrame(np.random.normal(size=[len(forecast_index), len(columns)]), index=forecast_index, columns=columns)
+    )
```

Comment:
This change is necessary because `TimeSeriesPredictor.fit()` currently fails if some time series in the training set contain of all NaNs.